autogpt_platform/backend/backend/server/v2/store/routes.py

```diff
@@ -620,6 +625,8 @@ def remove_credentials(obj):
                 del obj["credentials"]
             if "creds" in obj:
```

Comment:
This should probably be a greater utility function that gets called :) 

we can probably also strip anything that has secret = true